docs/configuration.md

```diff
@@ -1506,6 +1506,11 @@ Apart from these, the following properties are also available, and may be useful
     of this setting is to act as a safety-net to prevent runaway uncancellable tasks from rendering
     an executor unusable.
   </td>
+  <td><code>spark.stage.maxAttempts</code></td>
```

Comment:
@squito maxConsecutiveAttempts? Realized maxAttempts is slightly misleading / might be good to emphasize that they're consecutive